homeassistant/helpers/llm.py

```diff
@@ -677,6 +677,19 @@ def on_homeassistant_close(event: Event) -> None:
 
                 self.parameters = vol.Schema(schema)
 
+                aliases: list[str] = []
+                if entity_entry.name:
+                    aliases.append(entity_entry.name)
```

Comment:
The entry name is already provided as the tool name, no? Should it still be included in aliases?

Reply:
No, the script name is based on entity_id, and the user can set a different name from the entity configuration dialog